docs/community/community-call.md

```diff
@@ -0,0 +1,52 @@
+---
+id: community-call
+title: Update Call for SAP Community
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Community Call
+description: SAP Cloud SDK hosted community calls to share updates and demonstrate best practices for developers.
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+image:
+---
+
+## Community Call
+
+https://community.sap.com/resources/community-calls
+
+## Topics Backlog
+
+* Build an application usind SAP Cloud SDK in SAP Bussiness Application Studio
+
+## Upcoming Dates 2020
+
+* May 5 - Consuming OData v4 Services using SAP Cloud SDK
+* Jun 2
+* Jul 7
+* Aug 4
+* Sep 1
+* Oct 6
+* Nov 3
+* Dec 1
+
+## History
+
+### 2020
+
+* Apr 7 - SAP Cloud SDK for JavaScript goes open source
+* Mar 4 - Get from Zero to Production with SAP Cloud SDK
```

Comment:
Some of those titles may not be self-explanatory, so we should add a description / agenda